eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java

```diff
@@ -48,7 +50,9 @@
     
     private UserController userService;
 
-    private BeanManager beanManager;    
+    private BeanManager beanManager;
```

Comment:
\[Checkstyle\] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2\.